### PR TITLE
profiles/base: stable.use.mask dev-java/fop[test]

### PR DESCRIPTION
**Pull request by vaukai**
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

**Comment by Flowdalic** [comment]
Shouldn't we simply stabilize mockito-4.4.0 instead of adding the mask? Or am I missing something?

**Comment by vaukai** [comment]
> Shouldn't we simply stabilize mockito-4.4.0 instead of adding the mask? Or am I missing something?

It's blocked by [libffi](https://bugs.gentoo.org/showdependencytree.cgi?id=837749&hide_resolved=1)

File: profiles/base/package.use.stable.mask [hunk 1]
```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-05-03)
+# Test dep mockito:4 is not stable. Bug #727504.
```
Comment (Flowdalic) [inline profiles/base/package.use.stable.mask:8]:
```suggestion
# fop[test] depends on mockito:4, which is not yet stable (bug #727504),
# and itself blocked by libffi stabilization (bug #813756).
```

Comment (vaukai) [reply profiles/base/package.use.stable.mask]: Thanks for wording. Changes done.

**Pull request closed without merging**